src/Confirm.js

```diff
@@ -4,9 +4,9 @@ var { Button, Modal } = require('react-bootstrap');
 var Confirm = React.createClass({
     propTypes: {
         body: React.PropTypes.any.isRequired,
-        buttonText: React.PropTypes.string,
-        cancelText: React.PropTypes.string,
-        confirmText: React.PropTypes.string,
+        buttonText: React.PropTypes.any,
```

Comment:
Yes

On Thu, Jun 2, 2016, 7:09 AM gbhrdt <notifications@github.com> wrote:

> In src/Confirm.js
> <https://github.com/gregthebusker/react-confirm-bootstrap/pull/3#discussion_r65545896>
> :
>
> > @@ -4,9 +4,9 @@ var { Button, Modal } = require('react-bootstrap');
> >  var Confirm = React.createClass({
> >      propTypes: {
> >          body: React.PropTypes.any.isRequired,
> > -        buttonText: React.PropTypes.string,
> > -        cancelText: React.PropTypes.string,
> > -        confirmText: React.PropTypes.string,
> > +        buttonText: React.PropTypes.any,
>
> Shouldn't we change the type of body as well then?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/gregthebusker/react-confirm-bootstrap/pull/3/files/1f44e4941581ce5c1a1e6defd14ceb6c3e02ab0d#r65545896>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACZ0uSRH3w879_cuvPKjA6Qnhcrc76hks5qHuQcgaJpZM4IsgoU>
> .
>